crates/cheatcodes/spec/src/vm.rs

```diff
@@ -287,6 +287,10 @@ interface Vm {
     #[cheatcode(group = Evm, safety = Unsafe)]
     function loadAllocs(string calldata pathToAllocsJson) external;
 
+    /// Clones a source account code, state, balance and nonce to a target account and updates in-memory revm state.
```

Comment:
```suggestion
    /// Clones a source account code, state, balance and nonce to a target account and updates in-memory EVM state.
```

Above too

Reply:
yep, I included changes in https://github.com/foundry-rs/foundry/pull/9048/commits/77e0033c5aab132819b24b179cadc8dc6d4eb865 to save a CI failed run due to cargo cheats